src/main/java/org/apache/rocketmq/dashboard/aspect/admin/MQAdminAspect.java

```diff
@@ -57,19 +54,11 @@ public Object aroundMQAdminMethod(ProceedingJoinPoint joinPoint) throws Throwabl
         long start = System.currentTimeMillis();
         Object obj = null;
         try {
-            MethodSignature signature = (MethodSignature)joinPoint.getSignature();
-            Method method = signature.getMethod();
-            MultiMQAdminCmdMethod multiMQAdminCmdMethod = method.getAnnotation(MultiMQAdminCmdMethod.class);
-            if (multiMQAdminCmdMethod != null && multiMQAdminCmdMethod.timeoutMillis() > 0) {
-                MQAdminInstance.initMQAdminInstance(multiMQAdminCmdMethod.timeoutMillis(),rmqConfigure.getAccessKey(),rmqConfigure.getSecretKey(), rmqConfigure.isUseTLS());
-            }
-            else {
-                MQAdminInstance.initMQAdminInstance(0,rmqConfigure.getAccessKey(),rmqConfigure.getSecretKey(), rmqConfigure.isUseTLS());
-            }
+            MQAdminInstance.createMQAdmin(mqAdminExtPool);
```

Comment:
in case of race condition, MQAdminExt mqAdminExt can not be a static value, may be we should new MQAdminInstance here.